csvViewer/file_manager.rb

```diff
@@ -0,0 +1,24 @@
+require('./reader.rb')
+module FILEMANAGER
```

Comment:
```suggestion
class FILEMANAGER
```
use class not module and don't use self, we don't need to call class method